python/tk_multi_reviewsubmission/actions.py

```diff
@@ -0,0 +1,167 @@
+# Copyright (c) 2013 Shotgun Software Inc.
+#
+# CONFIDENTIAL AND PROPRIETARY
+#
+# This work is provided "AS IS" and subject to the Shotgun Pipeline Toolkit
+# Source Code License included in this distribution package. See LICENSE.
+# By accessing, using, copying or modifying this work you indicate your
+# agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
+# not expressly granted therein are reserved by Shotgun Software Inc.
+
+import sgtk
+import os
+import copy
+
+from .submitter_create import CreateSubmitter
+from .submitter_sgtk import SGTKSubmitter
+
+
+class Actions(object):
+    def __init__(self):
+        """
+        Construction
+        """
+        self.__app = sgtk.platform.current_bundle()
+        self.submit_with_create = self.__app.get_setting("submit_using_create")
+
+        if self.submit_with_create:
```

Comment:
@robblau and I discussed this flag and we both agreed that the Create implementation should be the only one moving forward. The Create workflow is much better than what we had before.

Nobody will miss the old feature because nobody uses it.